scipy/stats/tests/test_distributions.py

```diff
@@ -9,6 +9,7 @@
 from pathlib import Path
 import os
 import json
+import numpy as np
```

Comment:
This import is duplicate from
https://github.com/bsdz/scipy/blob/f85673cc58af28cbb924d7d2929c447c7d655c72/scipy/stats/tests/test_distributions.py#L21-L22
I'd imagine it can be removed without issue